### PR TITLE
miscellaneous scripts 

### PR DESCRIPTION
**Pull request by Keyurpatel93**
…keys exist in the templates metadata

roi.cpp copyOnCrop=True is now the default
rotate.cpp rotates a mat by 90 intervals
extractvaluefromLlist.cpp extracts a single value from a list in metadata and saves it in a new key
majorityvoting.cpp Calculates which class has the highest scores when given a list of scores for each class
scorelevelfusion.cpp Fuses the scores together for each class when given a list of scores for each class

### CODE REVIEW

File: openbr/plugins/classification/caffe.cpp [hunk 1]
```diff
@@ -2,7 +2,7 @@
 #include <openbr/core/opencvutils.h>
 
 #include <caffe/caffe.hpp>
-#include <caffe/layers/memory_data_layer.hpp>
+//#include <caffe/layers/memory_data_layer.hpp>
```
Comment (jklontz) [inline openbr/plugins/classification/caffe.cpp]: This change shouldn't be comitted

**Comment by jklontz** [comment]
LGTM otherwise

**Comment by Keyurpatel93** [comment]
Uncommented out specified line https://github.com/biometrics/openbr/pull/515/commits/dc67cf8cff88aa217be2e21f2c23b240b042d003

**Comment by jklontz** [comment]
Thanks!

**Pull request merged**